python/lsst/meas/algorithms/normalizedCalibrationFlux.py

```diff
@@ -253,18 +287,23 @@ def _measure_aperture_correction(self, exposure, catalog):
             ).apCorrMap
 
             ap_corr_field = ap_corr_map.get(raw_name + "_instFlux")
-        except MeasureApCorrError:
-            self.log.warning("Failed to measure full aperture correction for %s", raw_name)
+        except MeasureApCorrError as e:
+            self.log.warning("Failed to measure full aperture correction for %s with the following error %s",
+                             raw_name, e)
 
-            sel = self.fallback_source_selector.run(catalog, exposure=exposure).selected
-            sel &= (~catalog[self.config.raw_calibflux_name + "_flag"]
-                    & ~catalog[self.config.measure_ap_corr.refFluxName + "_flag"])
+            initSel = self.fallback_source_selector.run(catalog, exposure=exposure).selected
+            sel = (initSel & ~catalog[self.config.raw_calibflux_name + "_flag"]
+                   & ~catalog[self.config.measure_ap_corr.refFluxName + "_flag"])
 
             n_sel = sel.sum()
 
             if n_sel == 0:
```

Comment:
This can be a walrus:

```suggestion
            if (n_sel := sel.sum()) == 0:
```